src/Behat/Behat/Hook/Call/RuntimeFeatureHook.php

```diff
@@ -40,10 +42,19 @@ public function __construct($scopeName, $filterString, $callable, $description =
         parent::__construct($scopeName, $filterString, $callable, $description);
 
         if ($this->isAnInstanceMethod()) {
+            if (is_array($callable)) {
```

Comment:
Depending on the type of callable that we have we get the class name and method name using one way or another